### ISSUE TITLE
Consider setting delaybeforesend to None

### ISSUE THREAD

**Issue by shoyer** [blink1073/oct2py#109]
[pexpect's `spawn`](http://pexpect.readthedocs.io/en/stable/api/pexpect.html#spawn-class) has a `delaybeforesend` parameter that defaults to 50ms. This adds about 300 ms of overhead per MATLAB command when using oct2py:
```
In [41]: from oct2py.speed_check import SpeedCheck

In [42]: test = SpeedCheck()
May not be able to display plots properly without gnuplot, please install it

In [43]: test.array = np.arange(2)

In [44]: %timeit test.large_array_put()
1 loops, best of 3: 307 ms per loop

In [45]: %timeit test.large_array_get()
1 loops, best of 3: 307 ms per loop

In [46]: test.octave._session.engine.repl.child.delaybeforesend = None

In [47]: %timeit test.large_array_put()
100 loops, best of 3: 4.44 ms per loop

In [48]: %timeit test.large_array_get()
100 loops, best of 3: 3.68 ms per loop
```

**Comment by blink1073** [comment]
Nice catch!  I'll make the change in octave_kernel so it benefits both libs.

**Comment by blink1073** [comment]
Released in `octave_kernel@0.24.8.`.  Thanks!
